metals/src/main/scala/scala/meta/internal/builds/MavenBuildTool.scala

```diff
@@ -19,16 +20,18 @@ case class MavenBuildTool(userConfig: () => UserConfiguration)
     AbsolutePath(out)
   }
 
+  private lazy val bloopMavenPluginVersion = BuildInfo.mavenBloopVersion
```

Comment:
I think it's ok, but I just want to check. This version in the past _could_ be changed by the version of Bloop that the user configured. Now that changes since it's no longer tied to it. That's fine right? We should just always use the latest for the integrations?